nursery/check-thread-suspend-count-exceeded.yml

```diff
@@ -0,0 +1,27 @@
+rule:
+  meta:
+    name: check thread suspend count exceeded
+    authors:
+      - ervinocampo@google.com
+    scopes:
+      static: file
```

Comment:
could the scope be `function`?

Reply:
Thanks for reviewing the PR Willi. I opted for a file scope since the instructions being matched for "create thread bypass process freeze" will typically be on a separate function as the rest of the instructions being matched. For the simplest implementation of this, the call to NtCreateThreadEx parameter will be on the main function, while the call to NtSuspendThread will be on the function passed to NtCreateThreadEx.